### ISSUE TITLE
Bug: (sort-imports) prefix-only builtin modules are defined as external modules

### ISSUE THREAD

**Issue by Wondermarin** [azat-io/eslint-plugin-perfectionist#65]
### Describe the bug

When using the `node:` prefix for an `builtin` module, the `sort-imports` rule defines it as an `external` module.

### Code example

**IN CONFIG:**
```json
"perfectionist/sort-imports": [
  "error",
  {
    "type": "natural",
    "groups": [
      "builtin",
      "external",
      "internal",
      "parent",
      "sibling",
      "index",
      "side-effect"
    ]
  }
]
```

**IN CODE:**
```js
import { writeFile } from "node:fs/promises"; // ⤸ There is no space between groups.
import { createRoot } from "react-dom/client";
```

### ESLint version

v8.48.0

### ESLint Plugin Perfectionist version

v2.0.0

### Validations

- [X] Read the [docs](https://eslint-plugin-perfectionist.azat.io).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by Wondermarin** [comment]
Made [PR](https://github.com/azat-io/eslint-plugin-perfectionist/pull/69).

**Comment by azat-io** [comment]
Thanks! Released in [v2.0.1](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v2.0.1)
